modules/auxiliary/scanner/http/jira_user_enum.rb

```diff
@@ -0,0 +1,90 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Scanner
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Report
+  include Msf::Auxiliary::AuthBrute
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Jira Users Enumeration',
+        'Description' => %q{
+          This module exploits an information disclosure vulnerability that allows an
+          unauthenticated user to enumerate users in the /ViewUserHover.jspa endpoint.
+          This only affects Jira versions < 7.13.16, 8.0.0 ≤ version < 8.5.7, 8.6.0 ≤ version < 8.12.0
+          Discovered by Mikhail Klyuchnikov @__mn1__
+          This module was only tested on 8.4.1
+        },
+        'Author' => [ 'Brian Halbach' ],
```

Comment:
It is a good idea to also give credits to who discovered the vulnerability or wrote the PoC, and add a comment saying who did what. Here is an example: https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/multi/fileformat/libreoffice_logo_exec.rb#L26-L29

Reply:
I updated the author info to be similar to https://github.com/rapid7/metasploit-framework/blob/master/modules/exploits/multi/fileformat/libreoffice_logo_exec.rb#L26-L29

Reply:
Thanks for updating this. There is a typo (missing comma) in the `Author` section, which makes our checks fail. Here is how it should be:
```ruby
        'Author' =>
          [
            'Brian Halbach', # msf module author
            'Mikhail Klyuchnikov' # initial discovery and PoC
          ],
```